### PR TITLE
Provide an introduction to the Canton Console

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
The purpose of this page is to expose developers to the availability of the Canton Console as part of the SDK to interact with the Sandbox, provide tips on how to consult the built-in documentation and provide further pointers to the in-depth documentation page part of the Canton user manual.

changelog_begin
changelog_end

### CODE REVIEW

File: docs/source/tools/canton-console.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+=========================
+
+One of the objects available in the Canton Console represents the Sandbox itself. The object is called
+``sandbox`` and you can use it to interact with the Sandbox. For example, you can list the DARs loaded
+on the Sandbox by running the following command::
+
+  sandbox.dars.list()
+
+Among the various features available as part of the Canton Console, you can manage parties and packages,
+check the health of the Sandbox, perform pruning operations and more. You are advised to consult the
```
Comment (carrielaben-da) [inline docs/source/tools/canton-console.rst:59]:
```suggestion
check the health of the Sandbox, perform pruning operations and more. Consult the
```

File: docs/source/tools/canton-console.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+built-in documentation as mentioned above to learn about further capabilities, as well as consulting
+the main documentation page for the Canton Console.
```
Comment (carrielaben-da) [inline docs/source/tools/canton-console.rst:61]:
```suggestion
built-in documentation mentioned above and the main documentation page for the Canton console to learn about further capabilities.
```

Is the main documentation page we refer to here https://docs.daml.com/2.5.0/index/operate-a-daml-ledger/canton-administration-quickstart/index.html?

File: docs/source/tools/canton-console.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Introduction
+============
+
+Canton offers a console where administrative or debugging commands can be run.
+
+When you run the Sandbox using ``daml start`` or ``daml sandbox``, you are effectively starting an
+in-memory instance of Canton with a single domain and a single participant.
+
+As such, you can interact with the running Sandbox using the Canton Console, just like you would
```
Comment (carrielaben-da) [inline docs/source/tools/canton-console.rst:15]:
```suggestion
As such, you can interact with the running Sandbox using the console, just like you would
```

File: docs/source/tools/canton-console.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+in a production environment.
+
+The purpose of this page is to give a few pointers on how the Canton Console can be used to
+interact with a running Sandbox. For an in-depth guide on how to use this tool against a production,
```
Comment (carrielaben-da) [inline docs/source/tools/canton-console.rst:19]:
```suggestion
The purpose of this page is to give a few pointers on how the console can be used to
interact with a running Sandbox. For an in-depth guide on how to use this tool against a production,
```

File: docs/source/tools/canton-console.rst [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+staging or testing environment, consult the main documentation for the Canton Console.
```
Comment (carrielaben-da) [inline docs/source/tools/canton-console.rst:20]:
```suggestion
staging or testing environment, consult the main documentation for the Canton console.
```
We should have a link here; see comment below.

File: docs/source/tools/canton-console.rst [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+Run the Canton Console against the Sandbox
+==========================================
+
+Once you have a Sandbox running locally (for example after running ``daml start`` or ``daml sandbox``)
+you can start the Canton Console with the following command (in a separate terminal from the one where
```
Comment (carrielaben-da) [inline docs/source/tools/canton-console.rst:26]:
```suggestion
you can start the console with the following command (in a separate terminal from the one where
```

File: docs/source/tools/canton-console.rst [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+   daml canton-console
+
+Once the Canton Console started (it might take some time the first time) you can quit the session by
+running the ``exit`` command.
+
+Built-in documentation
+======================
+
+The Canton Console comes with built-in documentation that allows you to discover its features. You
```
Comment (carrielaben-da) [inline docs/source/tools/canton-console.rst:37]:
```suggestion
The Canton console comes with built-in documentation that allows you to discover its features. You
```

File: docs/source/tools/canton-console.rst [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+  health.help
+
+Or go more in depth about specific items within that object as in the following example::
+
+  health.help("status")
+
+Interact with the Sandbox
+=========================
+
+One of the objects available in the Canton Console represents the Sandbox itself. The object is called
```
Comment (carrielaben-da) [inline docs/source/tools/canton-console.rst:52]:
```suggestion
One of the objects available in the Canton console represents the Sandbox itself. The object is called
```

File: docs/source/tools/canton-console.rst [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+``sandbox`` and you can use it to interact with the Sandbox. For example, you can list the DARs loaded
+on the Sandbox by running the following command::
+
+  sandbox.dars.list()
+
+Among the various features available as part of the Canton Console, you can manage parties and packages,
```
Comment (carrielaben-da) [inline docs/source/tools/canton-console.rst:58]:
```suggestion
Among the various features available as part of the console, you can manage parties and packages,
```

**Review by carrielaben-da — APPROVED** [review]
Mostly small things. We use console as a common noun exclusively in the existing docs so I tried to make that consistent here.

File: docs/source/tools/canton-console.rst (hunk 6 above)
Comment (stefanobaghino-da) [reply docs/source/tools/canton-console.rst]: Addressed in https://github.com/digital-asset/daml/pull/15292/commits/f2f3f3bfb8c5e4ebc0c8c539606e52f83b98d132

File: docs/source/tools/canton-console.rst (hunk 7 above)
Comment (stefanobaghino-da) [reply docs/source/tools/canton-console.rst]: Addressed in https://github.com/digital-asset/daml/pull/15292/commits/f2f3f3bfb8c5e4ebc0c8c539606e52f83b98d132

File: docs/source/tools/canton-console.rst (hunk 2 above)
Comment (stefanobaghino-da) [reply docs/source/tools/canton-console.rst]: Addressed in https://github.com/digital-asset/daml/pull/15313

**Comment by garyverhaegen-da** [comment]
A bit late to the party here, but out of curiosity: any reason this is in the daml repo and not the canton repo?

**Pull request merged**